actioncable/test/connection/cross_site_forgery_test.rb

```diff
@@ -53,6 +53,13 @@ def send_async(method, *args)
     assert_origin_not_allowed "http://rails.co.uk"
   end
 
+  test "allow same origin as host" do
+    @server.config.allow_same_origin_as_host = true
+    assert_origin_allowed "http://#{HOST}"
+    assert_origin_not_allowed "http://hax.com"
+    assert_origin_not_allowed "http://rails.co.uk/"
```

Comment:
I think the allowed one should be a literal string.. probably. And this last line needs to lose the trailing `/` to have much meaning.

Also, is this test leaving the config option set? The teardown makes sure to reset the others.